tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Repository/OdsRepositoryTests.cs

```diff
@@ -124,18 +127,33 @@ public static void GetBuyerOrganisationByOdsCode_WithInternalServerErrorResponse
         }
 
         [Test]
+        [Ignore("Fails intermittently on server")]
```

Comment:
If we're skipping the test for now would it make sense to leave it as it was?

Reply:
Reverted changess